CleverPush/Source/CPAppBannerModuleInstance.m

```diff
@@ -496,8 +494,9 @@ - (BOOL)bannerTargetingAllowed:(CPAppBanner*)banner {
             }
 
```

Comment:
there are no null check for "fromVersion"/to here, only for compareATT, but in android we do it for compareATT and "fromValue"/to